<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatorDlg</class>
 <widget class="QDialog" name="CreatorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>103</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>124</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Component Creator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="compLabel">
         <property name="text">
          <string>Com&amp;ponent:</string>
         </property>
         <property name="buddy">
          <cstring>compComboBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="compComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>559</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="savePushButton">
         <property name="text">
          <string>Sa&amp;ve to File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="storePushButton">
         <property name="text">
          <string>St&amp;ore in DB</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="applyPushButton">
         <property name="text">
          <string>Only C&amp;reate</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelPushButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>compComboBox</tabstop>
  <tabstop>savePushButton</tabstop>
  <tabstop>storePushButton</tabstop>
  <tabstop>applyPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
